input/pagecontent/reference.md

```diff
@@ -352,11 +350,19 @@ In the following, the various types of path references are discussed. Some examp
 
 The path to a top-level element is denoted by the element's name. Because paths are used within the context of a FSH definition or instance, the path MUST NOT include the resource name. For example, when defining a profile of Observation, the path to Observation.code is just `code`.
 
+**Example:**
+
+* The path to the status of a MedicationRequest:
+
+  ```
+  status
+  ```
+
```

Comment:
It seemed like, for consistency, we should have an example.  And it makes sense to introduce the simplest example before moving on to more complex examples.